app/services/actions/foreman/puppet_class/import.rb

```diff
@@ -0,0 +1,30 @@
+module Actions
+  module Foreman
+    module PuppetClass
+      class Import < Actions::EntryAction
+        def resource_locks
+          :import_puppetclasses
+        end
+
+        def run
+          output[:changed] = input[:changed]
```

Comment:
One should be able to use the input from the task as well in additional tasks. The "results" thing looks strange, looking forward to look at it personally on Monday :)